base-math/truncddsf.c

```diff
@@ -40,14 +43,26 @@ CONVERT_WRAPPER(
 	int exp, sexp;
 	
 	a_norm = getmantandexpd64 (a, &exp, 16, 1e16DD);
-	
+
```

Comment:
Thanks for removing the other occurrences of this change.
This one slipped in...  Can you remove it, too?

Reply:
Oops. I will remove it.